src/app/services/endpoints.service.ts

```diff
@@ -97,6 +97,13 @@ export class EndpointsService {
     return `${this.participants}participant_team/${teamId}`;
   }
 
+  /**
+   * Delete Member Participant
+   */
+  participantMemberURL(teamId) {
+    return `${this.participants}remove_member_from_participant_team/${teamId}`;
```

Comment:
I checked the participants url file and there is no "remove_member_from_participant_team" Because of which you are recieving not found error

Reply:
> @Kajol-Kumari I checked the participants url file and there is no "remove_member_from_participant_team" Because of which you are recieving not found error

@Suryansh5545  can you tell me which url file you are taking about?

Reply:
https://github.com/Cloud-CV/EvalAI/blob/master/apps/participants/urls.py

Reply:
Did you get it?

Reply:
yes @Suryansh5545 got it. Thanks!